docs/docs/guide/08-loan/07-cancel.md

```diff
@@ -0,0 +1,71 @@
+# Cancel a loan
+
+As a borrower, you have the option to cancel a loan you have created if you no
+longer want to proceed with it. However, this action is only possible if the
+loan's current status is marked as "requested".
+
+If you decide to cancel the loan, the collateral tokens that were being held as
+security for the loan will be transferred back to your account from the module
+account. This means that you will regain possession of the collateral tokens you
+had originally put up for the loan.
+
+```go title="x/loan/keeper/msg_server_cancel_loan.go"
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"loan/x/loan/types"
+)
+
+func (k msgServer) CancelLoan(goCtx context.Context, msg *types.MsgCancelLoan) (*types.MsgCancelLoanResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	loan, found := k.GetLoan(ctx, msg.Id)
+	if !found {
+		return nil, sdkerrors.Wrap(sdkerrors.ErrKeyNotFound, fmt.Sprintf("key %d doesn't exist", msg.Id))
+	}
+	if loan.Borrower != msg.Creator {
+		return nil, sdkerrors.Wrap(sdkerrors.ErrUnauthorized, "Cannot cancel: not the borrower")
+	}
+	if loan.State != "requested" {
+		return nil, sdkerrors.Wrapf(types.ErrWrongLoanState, "%v", loan.State)
+	}
+	borrower, _ := sdk.AccAddressFromBech32(loan.Borrower)
+	collateral, _ := sdk.ParseCoinsNormalized(loan.Collateral)
+	k.bankKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleName, borrower, collateral)
```

Comment:
Missing err check

Reply:
https://github.com/ignite/cli/pull/3352/commits/5790ff5ca986827fcbf33822ef563d5aec0524a3

Reply:
This time it should be `err :=` not `err =`. 
Note that you can run `make format` to format the code inside snippets.

Reply:
https://github.com/ignite/cli/pull/3352/commits/d0ef3e222f4c39baf65e36cdb3ed8d578620bc7a

Thanks for the `make format` tip!